### PR TITLE
white text into black text

### PR DESCRIPTION
**Pull request by ayusharma**
This PR removes the bug on https://biojsnet.herokuapp.com/starters/

screen_shot_2016-04-07_at_2 02 19_am

@jessica-jordan Please take a look.

### CODE REVIEW

**Comment by daviddao** [comment]
@jessica-jordan can you check?

**Comment by DennisSchwartz** [comment]
Thanks for all your work @ayusharma!

Maybe I'm doing sth wrong, but this looks like this on my end now:

screen shot 2016-04-07 at 21 34 22

File: content/themes/biojs-theme/assets/sass/layout.scss [hunk 1]
```diff
@@ -161,7 +161,7 @@ nav {
   min-height: 380px;
   line-height: 0;
   height: auto;
-  color: rgba(255,255,255,.9);
+  // color: rgba(255,255,255,.9);
```
Comment (jayjayjpg) [inline content/themes/biojs-theme/assets/sass/layout.scss]: @ayusharma As this class `.parallax-container` is a pretty global one used on many several instances on the website, I wouldn't change this css declaration. Instead, you can have a look at the `.hbs` templates of this bug  and try to find out if you might be able to fix the issue by applying one of our [text-coloring helper classes](https://github.com/biojs/biojs.net2/blob/master/content/themes/biojs-theme/assets/sass/main.scss)
This way you should be able to fix this one local text coloring bug without having to check every other instance of the `.parallax-container` class on our website for correct text coloring.

File: content/themes/biojs-theme/partials/tip-section.hbs [hunk 2]
```diff
@@ -2,11 +2,11 @@
 <div class="container">
 <div class="row">
 <div class="col s12 m4">
-  <div class="card biojs-dark-grey biojs-white-text">
-    <div class="card-content white-text">
```
Comment (jayjayjpg) [inline content/themes/biojs-theme/partials/tip-section.hbs:6]: @ayusharma Yes, good one! Can you also check if you need to apply another helper to fix the bug?

**Comment by ayusharma** [comment]
@jessica-jordan Please take a look at PR. I have undo the layout.scss changes. only changed in .hbs file.

**Comment by jayjayjpg** [comment]
Looking good. Many thanks!

**Pull request merged**